client/components/date-range/use-shortcuts.ts

```diff
@@ -57,13 +56,6 @@ export const getShortcuts = createSelector(
 				endDate: siteTodayStr,
 				period: DATERANGE_PERIOD.DAY,
```

Comment:
Are we able to change the period to `hour` now?

Reply:
Nice catch! 👍🏼

Reply:
Updated in [0069379](https://github.com/Automattic/wp-calypso/pull/98160/commits/006937994981bb1f38d0eec6c0e118f8237c2560).

Reply:
We may still need to keep [the redirection](https://github.com/Automattic/wp-calypso/blob/03273db9e018368e8ac2c5953485e753f70d0652/client/my-sites/stats/site.jsx#L402-L405) for choosing a single day on the date range picker.